modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -3,11 +3,35 @@
 // * monitoring/cluster_monitoring/metering-configure-hive-metastore.adoc
 
 [id="metering-use-mysql-or-postgresql-for-hive_{context}"]
-= Use MySQL or PostgreSQL for the Hive metastore
+= To use MySQL or PostgreSQL for the Hive metastore
```

Comment:
I think this would be better as "**Using** MYSQL or PostgreSQL for Hive metastore"